core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -270,6 +270,11 @@
           null,
           OperandTypes.EXACT_NUMERIC_EXACT_NUMERIC);
 
+  /**
+   * Casting operator used by PostgreSQL '<code>::</code>'.
+   */
+  public static final SqlBinaryOperator PG_CAST = new PostgreSQLCastOperator();
+
```

Comment:
Put it in the `SqlLibraryOperators` because it does not belongs to sql standard.